### PR TITLE
Consolidated REST and SSE under API section

### PR DESCRIPTION
**Pull request by psavkar**

### CODE REVIEW

File: docs/event-sources/README.md [hunk 1]
```diff
@@ -2,19 +2,25 @@
 
 **Event sources turn any API into an event stream. They can also turn any event stream into an API**.
 
-Sources collect data from services like Github, Stripe, the bitcoin blockchain, RSS feeds, and more. They emit new events produced by the service, which you can consume using Pipedream's APIs.
+Event sources run on Pipedream's infrastructure and collect data from services like Github, Stripe, the bitcoin blockchain, RSS feeds, and more. They emit new events produced by the service, which you can inspect in the UI or consume using real-time or batch APIs.
```
Comment (dylburger) [inline docs/event-sources/README.md]: I'd link to the REST and SSE API where you say "real-time" and "batch"

File: docs/.vuepress/config.js [hunk 2]
```diff
@@ -105,9 +104,9 @@ module.exports = {
         children: ["/cli/install/", "/cli/login/", "/cli/reference/"]
       },
       {
-        title: "REST API",
```
Comment (dylburger) [inline docs/.vuepress/config.js]: API -> APIs for header

File: docs/api/rest/README.md [hunk 3]
```diff
@@ -1,15 +1,10 @@
-# API Reference
+# REST
```
Comment (dylburger) [inline docs/api/rest/README.md]: I'd change this to "REST API"

File: docs/event-sources/README.md (hunk 1 above)
Comment (psavkar) [reply docs/event-sources/README.md]: done

File: docs/.vuepress/config.js (hunk 2 above)
Comment (psavkar) [reply docs/.vuepress/config.js]: done

File: docs/api/rest/README.md (hunk 3 above)
Comment (psavkar) [reply docs/api/rest/README.md]: done

**Pull request merged**